### DISCUSSION TITLE
Pyrolysis models

### DISCUSSION THREAD

**Discussion by Youcef05122000** [firemodels/fds#13239] [Q&A] [answered]
Hello,

I have a question about the use of pyrolysis. I want to model a fire without adding the heat release rate (HRR) of the fire, using only pyrolysis and the decomposition of the solid. I tried an example with a PMMA plate, where I placed a heat source near the plate. FDS showed the decomposition of the solid over time,  but there was no fire production.

I would like to know if it is possible to generate a fire from the pyrolysis of a solid. If so, how can I achieve this?

Thank you for your help.

**Comment by rmcdermo** [comment]
I suspect that you did not add a gas phase reaction.  Have a look at this example.

https://github.com/firemodels/fds/blob/master/Verification/Fires/couch.fds

**Reply by Youcef05122000** [reply]
Hello,

I tried what you suggested. I used the same code as couch.fds,  but only changed the burning material to PMMA. However, I did not get the same result; there was no fire. 
Could you please take a look at my code and tell me what I should change to get the same fire effect using PMMA?
[couch_pmma.txt](https://github.com/user-attachments/files/16443314/couch_pmma.txt)

Thank you.

**Reply by mcgratta** [reply]
You cannot assume that any material will sustain a fire, regardless of its properties.

**Reply by rmcdermo** [reply, marked as answer]
Did you successfully run the case before your added PMMA?

**Reply by drjfloyd** [reply]
You have changed the cover material of the sofa from fabric to PMMA.

Do you know what temperature your PMMA material reaction starts to create PMMA gas?
Have you checked if the ignition source is causing your PMMA to reach that temperature?

**Reply by Youcef05122000** [reply]
Yes, I did, and it worked correctly; I had the same results as shown in the FDS User Guide. I faced problems when I changed the burning material with PMMA.

**Reply by Youcef05122000** [reply]
thank you for all, i have found the solution.

**Reply by Youcef05122000** [reply]
Hello, 

I'm working on a model of pyrolysis fire in a confined and under-ventilated compartment. I have tried the pyrolysis model that I showed you before. I achieved a fire with an HRR of approximately 400 kW, but I want to achieve a fire of 1000 kW.

 I would like to know if there is a way to control the heat release rate produced by the pyrolysis of the material. Specifically, is there a parameter that we can change in the pyrolysis model of PMMA to achieve a higher HRR ?

Thank you.

**Reply by drjfloyd** [reply]
You have an Arrhenius reaction rate where the reaction rate is proportional to A e^(-E/RT).  What inputs for your MATL could you change to make A e^(-E/RT) larger?
